EmerisHQ/demeris
# fix: skip fee calculation for small amounts

## Description

In the swap widget if you enter an input value less than the chain fee value, it will turn zero and request fails.

Fixes https://allinbits.slack.com/archives/C02AXJ0K9F1/p1654434283026519?thread_ts=1654266547.117689&cid=C02AXJ0K9F1

## Feature flags

N/A

## Testing

- Select ATOM as the input asset and enter a small value like `0.0001`
- Select OSMO as output
- It should show the expected output value.
